pending/biglybt.xml

```diff
@@ -48,42 +59,53 @@
     <action command="delete" search="glob" path="$$profile$$\active\*.bak"/>
     <action command="delete" search="glob" path="$$profile$$\subs\*.biglybt.bak"/>
     <action command="delete" search="walk.all" path="$$programfiles$$\backup\"/>
+    <!--
+    BiglyBT AzConfigBackup not yet figured out! I will do this later! Tobias.
     <action command="delete" search="walk.all" path="$$programfiles$$\BiglyBT AzConfigBackup\"/>
+    -->
+    <!-- Delete *.bak recursive: -->
+    <action command="delete" search="walk.files" path="$$profile$$\" regex="\.[Bb][Aa][Kk]$"/>
```

Comment:
This recursive search looks right. Great.

Reply:
:-)